airflow/utils/log/es_task_handler.py

```diff
@@ -255,7 +256,9 @@ def close(self):
 
         # Mark the end of file using end of log mark,
         # so we know where to stop while auto-tailing.
-        self.handler.stream.write(self.end_of_log_mark)
+        if self.write_stdout:
+            print()
+        self.handler.emit(logging.makeLogRecord({'msg': self.end_of_log_mark}))
```

Comment:
pls review [AIRFLOW-5528] (end_of_log_mark should not be a log record (#6159) 
where @pingzh changed this line:
```diff
- self.handler.emit(logging.makeLogRecord({'msg': self.end_of_log_mark}))
+ self.handler.stream.write(self.end_of_log_mark)
```